masterlist.yaml

```
@@ -14435,6 +14637,12 @@ plugins:
       - crc: 0x6CE0A40A
         util: 'SSEEdit v4.0.3'
 
+  - name: 'Black Mage Armor SE.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/356' ]
```

This URL is missing a trailing slash:

```suggestion
    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/356/' ]
```